megamek/src/megamek/client/ui/swing/CommonSettingsDialog.java

```diff
@@ -1723,6 +1734,7 @@ protected void cancelAction() {
         planetaryConditionsShowLabels.setSelected(GUIP.getPlanetaryConditionsShowLabels());
         planetaryConditionsShowValues.setSelected(GUIP.getPlanetaryConditionsShowValues());
         planetaryConditionsShowIndicators.setSelected(GUIP.getPlanetaryConditionsShowIndicators());
+        planetaryConditionsBackgroundTransparency.setText(String.format("%d", GUIP.getPlanetaryConditionsBackgroundTransparency()));
```

Comment:
Isn't there an input control that's like a number spinner or something that ensures the user is limited to entering numbers? JSpinner?